web/src/billing/upgrade.ts

```diff
@@ -29,6 +29,10 @@ if ($("input[type=hidden][name=schedule]").length === 1) {
     // we need to override schedule from localstorage if it was set.
     selected_schedule = $<HTMLInputElement>("input[type=hidden][name=schedule]").val()!;
 }
+if (page_params.fixed_price !== null) {
+    // By default, we show annual schedule (and price) for a fixed-price plan.
+    selected_schedule = "annual";
+}
```

Comment:
Do we want this to override what was in local storage? Maybe it doesn't matter but it seems like the intent would be that we should just replace the default in the line `typeof ls_selected_schedule === "string" ? ls_selected_schedule : "monthly";` with defining a `default_selected_schedule` based on the plan type. Hmm, I guess that approach might be annoying if you went to /upgrade before your fixed-price plan code programmed. I think this is probably OK for now given that analysis.